cli/flox/src/utils/message.rs

```diff
@@ -37,3 +37,7 @@ pub(crate) fn package_installed(pkg: &PackageToInstall, environment_description:
         pkg.id
     ));
 }
+
+pub(crate) fn service_stopped(service_name: &str) {
+    updated(format!("Service '{}' stopped", service_name));
```

Comment:
nit: are we expecting this to be used more than once?

Reply:
What I really wanted was a function that says "give me a name and I'll print the message", and we already have a place for functions like that.